pimd/pim_vty.c

```diff
@@ -70,10 +70,6 @@ int pim_debug_config_write(struct vty *vty)
 		vty_out(vty, "debug igmp trace\n");
 		++writes;
 	}
-	if (PIM_DEBUG_IGMP_TRACE_DETAIL) {
```

Comment:
This macro and its associated flag still exist - wouldn't it be clearer just to remove all uses and use PIM_DEBUG_IGMP_TRACE ?

Reply:
yeah not quite ready to do that since the other command turns more on.  

PIM debugs are all kind of messed up and I'm not ready to take them on.  I just want to stop shooting myself in the foot in the mean time.

Reply:
ok, fair enough - you'll need that foot...